apps/frontend/src/app/tournament-admin/tournament-admin-container/tournament-admin.component.scss

```diff
@@ -38,4 +38,9 @@
 textarea {
   resize: none;
   font-size: 15px;
+  font-family: 'Roboto';
```

Comment:
A more sustainable solution than redefining the font-family at individual textareas would be to add this in your styles.scss:
```
textarea, input, button, select { font-family: inherit; font-size: inherit; }
```